app/models/alchemy/picture.rb

```diff
@@ -71,7 +71,7 @@ class Picture < ActiveRecord::Base
     }
 
     scope :deletable, -> {
-      where("#{self.table_name}.id NOT IN (SELECT picture_id FROM alchemy_essence_pictures)")
+      where("#{self.table_name}.id NOT IN (SELECT picture_id FROM #{EssencePicture.table_name})")
```

Comment:
We shouldn't use an dynamic string without escaping it here either. Please use the Hash or Array syntax.